doc/tutorial/gpu_data_convert.txt

```diff
@@ -4,19 +4,25 @@
 PyCUDA/CUDAMat/Gnumpy compatibility
 ===================================
 
+.. note::
+
+    The following applies to the old GPU backend in ``theano.sandbox.cuda``.
+    The new backend in ``theano.gpuarray`` directly uses the GPUArray
+    types in PyCUDA.
+
 PyCUDA
 ======
 
-Currently, PyCUDA and Theano have different objects to store GPU
-data. The two implementations do not support the same set of features.
-Theano's implementation is called *CudaNdarray* and supports
-*strides*. It also only supports the *float32* dtype. PyCUDA's implementation
-is called *GPUArray* and doesn't support *strides*. However, it can deal with
-all NumPy and CUDA dtypes.
+Theano's new GPU backend (:ref:`theano.gpuarray<libdoc_gpuarray>`) is based on 
+PyCUDA's GPUArray objects. However, if you are still using the old backend,
```

Comment:
For sure, GPUarray no longer uses CudaNdarray types. When I said "based on pycuda's objects", I meant that it's not the same, but shares most of the interface. I could just explicitely say compatible.